src/lib/config/ModelConfig.cc

```diff
@@ -910,6 +910,35 @@ void ModelConfig::postValidation() {
     invalid_ << "\t- Only a 'Flat' L1-Instruction-Memory Interface-Type is "
                 "supported. Interface-Type used is "
              << l1iType << "\n";
+
+  // Ensure LSQ-L1-Interface Load/Store Bandwidth is large enough to accomodate
```

Comment:
If we do this, then we can move all/some of this logic out of `postValidation` and into the appropriate `setValueBounds` call. 

I say some as there's an argument to be made about contextualising an errored lower bound with the current additions to `invalid_` (if we moved all logic into the `setValueBounds` call, then the user wouldn't be told the lower bound is x because the vector-length is y). There are arguments for both sides though to be honest. Pretty trivial to understand the lower bound when considering our target audience

Reply:
If we keep some logic here, then the additions to `invalid` should contain the values of `Vector-Length` and `Store-Bandwidth` rather than just stating the keys

Reply:
I get your point. The largest load that seems to crop up in RISC-V after a few tests seems to be 8 bytes (so a minimum lower bound being 8 bytes).

I still think due to the link between SVE/SME and Load/Store bandwidth, even though most users of SimEng _should_ know, the current error message helps. I think the current logic should stay, and the lower bound should be raised.

For RISC V, as above, this is 8 bytes. For aarch64 this is 16 bytes as SVE is always enabled at a minimum of 128 bits.

What are your thoughts in this case of keeping the logic, raising the `setValueBounds` to 8 bytes minimum, and a comment in the docs to explain why this minimum is needed?

Reply:
After a discussion in person with @jj16791, this was the chosen way forward and has been implemented in the latest commit.